README.md

```diff
@@ -51,10 +52,19 @@ You would want to run Zebra if you want to contribute to the
 Zcash network: the more nodes are run, the more reliable the network will be
 in terms of speed and resistance to denial of service attacks, for example.
 
-Zebra aims to be
-[faster, more secure, and more easily extensible](https://doc.zebra.zfnd.org/zebrad/index.html#zebra-advantages)
+Zebra aims to be [faster, more secure, and more easily extensible](https://doc.zebra.zfnd.org/zebrad/index.html#zebra-advantages)
 than other Zcash implementations.
 
+### Running Zebra with Docker
+
+You can easily run Zebra using Docker with a simple command
+
+```shell
+docker run -d zfnd/zebra:1.0.0-rc.0
+```
+
+For more information, read our [Docker documentation](book/src/user/docker.md).
```

Comment:
This seems to be a duplicate of the "Getting Started" section?
https://github.com/ZcashFoundation/zebra/pull/5504/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R82

Reply:
I think it makes more sense as a subheading of "Getting Started" than in the "About" section

Reply:
Fixed by [docs(docker): use existing getting started header](https://github.com/ZcashFoundation/zebra/pull/5504/commits/2a38204ee545d980e0afe3890daea7fabd2144d2)